cache/remotecache/gha/gha.go

```diff
@@ -33,20 +33,24 @@ func init() {
 }
 
 const (
-	attrScope   = "scope"
-	attrTimeout = "timeout"
-	attrToken   = "token"
-	attrURL     = "url"
-	version     = "1"
+	attrScope      = "scope"
+	attrTimeout    = "timeout"
+	attrToken      = "token"
+	attrURL        = "url"
+	attrRepository = "repository"
+	attrGHToken    = "ghtoken"
```

Comment:
needs docs updates in https://github.com/moby/buildkit?tab=readme-ov-file#github-actions-cache-experimental and https://docs.docker.com/build/cache/backends/gha/ to define these new attributes (cc @dvdksn)